### DISCUSSION TITLE
sqlite - setup

### DISCUSSION THREAD

**Discussion by peter233** [Kovah/LinkAce#411] [Support] [answered]
hello,

i have finally setup a linkaces docker instance and used the sqlite guide.
but unfortunately its not working.

![grafik](https://user-images.githubusercontent.com/34836355/156896061-04856ee1-41bb-4152-b570-60f1951ebadf.png)

need help to fix this.
chmod 0777 database.sqlite
![grafik](https://user-images.githubusercontent.com/34836355/156896131-99fd9913-2827-4a4d-b6f4-8da92ea645e2.png)

**Comment by Kovah** [comment, marked as answer]
Have you looked at #332 and #327?

**Comment by peter233** [comment]
big thx
